src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Setting `loadingPost` to true before the post action and resetting it to false upon failure is a good practice. However, you should ensure that `setLoadingPost(false)` is also called in case of any other exceptions or errors that might occur outside this block.
```diff
+    setLoadingPost(true);
+    if (!isEmptyMessageAllowed()) {
+      showMessage({
+        message: StringConstant.createPostFailedNoMessage,
+        type: 'danger'
+      });
+      setLoadingPost(false);
+      return true;
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->